controllers/svc/factory/factory_kube.go

```diff
@@ -86,52 +86,53 @@ func (this *KubeFactory) CreateDeployment() *apps.Deployment {
 					Labels: this.GetLabels(),
 				},
 				Spec: core.PodSpec{
-					Containers: []core.Container{{
-						Name: this.ctx.GetAppName().Str(),
-						Resources: core.ResourceRequirements{
-							Limits: core.ResourceList{
-								core.ResourceCPU:    resource.MustParse("1"),
-								core.ResourceMemory: resource.MustParse("1300Mi"),
-							},
-							Requests: core.ResourceList{
-								core.ResourceCPU:    resource.MustParse("500m"),
-								core.ResourceMemory: resource.MustParse("512Mi"),
+					Containers: []core.Container{
+						{
+							Name: this.ctx.GetAppName().Str(),
+							Resources: core.ResourceRequirements{
+								Limits: core.ResourceList{
+									core.ResourceCPU:    resource.MustParse("1"),
+									core.ResourceMemory: resource.MustParse("1300Mi"),
+								},
+								Requests: core.ResourceList{
+									core.ResourceCPU:    resource.MustParse("500m"),
+									core.ResourceMemory: resource.MustParse("512Mi"),
+								},
 							},
-						},
-						LivenessProbe: &core.Probe{
-							ProbeHandler: core.ProbeHandler{
-								HTTPGet: &core.HTTPGetAction{
-									Path: "/health/live",
-									Port: intstr.FromInt(8080),
+							LivenessProbe: &core.Probe{
```

Comment:
This seems to be a semantic difference, is it correct? Would you mind elaborating?